addon/components/tf-toggle-button.js

```diff
@@ -0,0 +1,82 @@
+/**
+  @module ember-ticketfly-toggle-button
+ */
+import Component from 'ember-component';
+import layout from '../templates/components/tf-toggle-button';
+import get from 'ember-metal/get';
+import set from 'ember-metal/set';
+import { tryInvoke } from 'ember-utils';
+import computed from 'ember-computed';
+
+/**
+  colorGroup options: 'neutral'
+  sizeGroup options: 'primary', 'secondary'
+
+  @public
+  @class TfToggleButton
+  @extends Ember.Component
+ */
+export default Component.extend({
+  layout,
+
+  tagName: 'button',
+  type: 'button',
+  attributeBindings: ['disabled', 'aria-label', 'aria-pressed', 'type'],
+  classNameBindings: ['buttonCategory', 'disabled:c-tf-toggle-button--disabled', '_pressed:c-tf-toggle-button--pressed'],
+  classNames: ['c-tf-toggle-button', 'u-pv-0', 'u-ph-xs'],
+
+  hook: 'tf-toggle-button',
+
+  colorGroup: '',
+  sizeGroup: 'primary',
+
+  buttonCategory: computed('colorGroup','sizeGroup', {
+    get() {
+      const sizeGroup = get(this, 'sizeGroup');
+
+      let buttonCategory = `c-tf-toggle-button--${sizeGroup}`;
+
+      const colorGroup = get(this, 'colorGroup');
```

Comment:
I think we should probably make these more single-purpose computed properties — so, `buttonCategory` is one and `colorCategory` is another.